go.mod

```diff
@@ -4,6 +4,7 @@ go 1.20
 
 require (
 	github.com/docker/docker-credential-helpers v0.7.0
+	github.com/need-being/go-tree v0.1.0
```

Comment:
Adding not a well known library such as need-being/go-tree is risky and since the amount of code we are taking about is less than 100 lines I would prefer that we we write our own.

Reply:
Since it is MIT, we can borrow their code.